jkube-kit/doc/src/main/asciidoc/inc/generator/_options_common.adoc

```diff
@@ -45,6 +45,10 @@ or already added by a generator which has been run previously.
 | *registry*
 | A optional Docker registry used when doing Docker builds. It has no effect for OpenShift S2I builds.
 | `jkube.generator.registry`
+
+| *tags*
+| A comma separated list of additional tags you want to tag your image with
+| `jkube.generator.tags`
```

Comment:
Reminder to create a new Issue:

These property names are misleading, they should be renamed to:
`jkube.generator.${generatorName}.tags` or applicable

Then add a note indicating that the `${generatorName}` placeholder should be changed

Reply:
Or does it work without the generator?

Reply:
The question was meant so that we tested it and create an issue, I'm not sure you did.
However, you have changed the documentation...

I'm sure that `jkube.generator.from` and others work when not specifying the generator name. We need to verify and make sure which properties need to be changed, and decide the approach.
**Please, provide feedback before doing any action.**

Reply:
You're right. In BaseGenerator there is a method `getConfigWithFallback` which checks system and project properties with given key for config field value:

https://github.com/eclipse/jkube/blob/0401297d4bcea783a94fc178af376c7a8a5a185d/jkube-kit/generator/api/src/main/java/org/eclipse/jkube/generator/api/support/BaseGenerator.java#L185

BaseGenerator config fields `name`, `alias`, `from`, `fromMode`, `registry` seems to be using `getConfigWithFallback` method so they seems to work with `jkube.generator` prefix without generator name.

`add` and `tags`(added in this PR) don't seems to be aligned with this approach. Right now they don't work with `jkube.generator` prefix and require generator name

Reply:
I've reverted my doc changes and created https://github.com/eclipse/jkube/issues/1489